src/FileSystem/FilesFinder.php

```diff
@@ -70,9 +70,16 @@ function (string $file): bool {
         // filtering files in directories collection
         $directories = $this->fileAndDirectoryFilter->filterDirectories($filesAndDirectories);
         $filteredFilePathsInDirectories = $this->findInDirectories($directories, $suffixes, $sortByName);
+        return [...$filteredFilePaths, ...$filteredFilePathsInDirectories];
```

Comment:
This function now returns a list of all possible files to analyse, without filtering out unchanged files